data/test/scenarios/wml_tests/UnitsWML/AbilitiesWML/test_overwrite_specials_filter.cfg

```diff
@@ -0,0 +1,112 @@
+#####
+# API(s) being tested: [damage]overwrite_specials=both_sides,[overwrite][filter_specials]
```

Comment:
There should be two separate tests - one for `[damage]overwrite_specials=both_sides` and one for `[overwrite][filter_specials]`.

Reply:
> There should be two separate tests - one for `[damage]overwrite_specials=both_sides` and one for `[overwrite][filter_specials]`.

[overwrite][filter_specials]. can't work without overwrite_specials attribute and like alice don't use [damage]apply_to=opponent, what overwtites_specials=both_sides or one_side is not pertinent in this test. Only  `[overwrite][filter_specials]` is realyy tested here.

Reply:
Alright, then the `API(s) being tested` line should be updated to show that.

Reply:
line changed